mcux/middleware/mcux-sdk-middleware-connectivity-framework/platform/connected_mcu/fwk_platform.c

```diff
@@ -188,54 +185,6 @@ void PLATFORM_GetMCUUid(uint8_t *aOutUid16B, uint8_t *pOutLen)
     return;
 }
 
-int PLATFORM_InitTimerManager(void)
```

Comment:
please do not have intermediate commits, squash the final changes to the prior commits